velox/exec/MemoryReclaimer.cpp

```diff
@@ -117,51 +136,66 @@ uint64_t ParallelMemoryReclaimer::reclaim(
           error(nullptr) {}
   };
 
-  std::vector<std::shared_ptr<AsyncSource<ReclaimResult>>> reclaimTasks;
-  for (const auto& candidate : candidates) {
-    if (candidate.reclaimableBytes == 0) {
-      continue;
+  uint64_t reclaimedBytes{0};
+  for (uint32_t i = 0, j = i; i < candidates.size(); i = j) {
```

Comment:
Why we need to handle priority in parallel memory reclaimer? thanks!

Reply:
Right, we don't need to. They all run at once.